File: python/ex26.py
soma = 0
num = 0
idosos = 0
maiores = 0
aux = 1
while aux > 0:
    idade = int(input())
    if idade < 0:
        aux = idade
    else:
        soma += idade
        num += 1
        if idade >= 18:
            maiores += 1
            if idade > 75:
                idosos += 1
media = soma / num
perc_idosos = (idosos / num) * 100
print(f'{media:.2f}')
print(f'{maiores}')
print(f'{perc_idosos:.2f}%')

    

File: c/ex26.c
//Lê idades, até ler uma inválida, imprime, média, qtd de maiores de idade e porcetagem de idosos
#include<stdio.h>
int main(){
    int idade = 1, qtd_geral = 0, qtd_maiores = 0, qtd_idosos = 0;
    float soma = 0;
    while(idade >= 0){
        scanf(" %i", &idade);
        if(idade >= 0){
            qtd_geral++;
            soma += idade;
            if(idade >= 18){
                qtd_maiores++;
                if(idade > 75){
                    qtd_idosos++;
                }
            }
        }
        else{
            if(qtd_geral == 0){
                qtd_geral++;
            }
        }
    }
    float media, pocentagem_idosos;
    media = soma / qtd_geral;
    pocentagem_idosos = ((qtd_idosos*1.0)/(qtd_geral))*100;
    printf("%.2f\n%i\n%.2f%%", media, qtd_maiores, pocentagem_idosos);
    return 0;
}